cmake/OpenCVFindCANN.cmake

```diff
@@ -5,6 +5,13 @@ if("cann${CANN_INSTALL_DIR}" STREQUAL "cann" AND DEFINED ENV{ASCEND_TOOLKIT_HOME
     message(STATUS "CANN: updated CANN_INSTALL_DIR from ASCEND_TOOLKIT_HOME=$ENV{ASCEND_TOOLKIT_HOME}")
 endif()
 
+if(EXISTS "${CANN_INSTALL_DIR}/opp/op_proto/built-in/inc")
+    set(LOW_CANN_VERSION "YES" )
```

Comment:
Can we use `CANN_VERSION_BELOW_6_3_ALPHA002` to make it meaningful?

Reply:
OK,this makes it easier to understand